### PR TITLE
feat(overrides): ignore (max|min)(xyz) in .xml

### PR DESCRIPTION
**Pull request by shmpwk**
In order to add unknown words below:
https://github.com/autowarefoundation/autoware.universe/runs/6571869093?check_suite_focus=true

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@shmpwk Could you look at other PRs and explain what each word means? :bow:

File: .cspell.json [hunk 1]
```diff
@@ -405,6 +407,7 @@
     "datafiles",
     "dataholder",
     "datastart",
+    "datatypes",
```
Comment (kenji-miyake) [inline .cspell.json:410]: I think this should be replaced by `data types`.

File: .cspell.json [hunk 2]
```diff
@@ -812,6 +820,9 @@
     "midnightblue",
     "milee",
     "mintcream",
+    "minx",
+    "miny",
+    "minz",
```
Comment (kenji-miyake) [inline .cspell.json:825]: These should be `{max,min}_{x,y,z}`.

**Comment by kenji-miyake** [comment]
Also, you can consider ignoring words in README files.
Related: https://github.com/autowarefoundation/autoware-documentation/pull/62#discussion_r839370965

File: .cspell.json (hunk 1 above)
Comment (shmpwk) [reply .cspell.json]: Yes, right! https://github.com/tier4/autoware-spell-check-dict/pull/280/commits/2590739fda75ef8df77d3fe911ea5e54419ebaa7

File: .cspell.json (hunk 2 above)
Comment (shmpwk) [reply .cspell.json]: They are written in `.xml` file.
I think it is better not to change those files.
So I changed ignoring `"(max|min)(xyz)"` in .xml files.

**Comment by shmpwk** [comment]
Thank you for your review.
I changed most words to be ignored in README file.

File: .cspell.json [hunk 3]
```diff
@@ -136,6 +136,7 @@
     {
       "filename": "**/{*.cpp,*.hpp,*.py,*.sh,*.xml,*.yaml,CMakeLists.txt}",
       "ignoreRegExpList": [
+        "(max|min)(xyz)",
```
Comment (kenji-miyake) [inline .cspell.json:139]: It's a good solution, but I think we have to encourage people to replace `minx` with `min_x`.
So I'm sorry, but could you close this PR and leave the min/max typos as they are? :pray:

File: .cspell.json (hunk 2 above)
Comment (kenji-miyake) [reply .cspell.json]: Thank you! Added another comment. https://github.com/tier4/autoware-spell-check-dict/pull/280#discussion_r880419583

File: .cspell.json (hunk 3 above)
Comment (shmpwk) [reply .cspell.json:139]: Ok!

**Comment by shmpwk** [comment]
Close by https://github.com/tier4/autoware-spell-check-dict/pull/280#discussion_r880419583

**Pull request closed without merging**